### ISSUE TITLE
Will there be wipe and backup option in bootui too ?

### ISSUE THREAD

**Issue by wazeeahmed** [chenxiaolong/DualBootPatcher#259]
Just hoping it might ; So that we would be able to backup and wipe for clean flash of a rom . As you can see aroma has touch issues so it's really difficult to use dual boot utility . Would be great if you implement it .

**Comment by chenxiaolong** [comment]
Unfortunately, this can't be easily done. The necessary tools are not available this early in the boot process.

**Comment by wazeeahmed** [comment]
thats so sad . Really thought that this would be possible by now . Anyways no sweat . Well at least you can add then in the dual boot utility zip . I mean backup function can at least be added there ? can't it ?

**Comment by webmaster33** [comment]
Yeah, patching, backup, restore, wipe would all features really useful. Hoping, that sometime there will be enough time to implement these features. 

At the moment I would be satisfied, if patching would be possible in a recovery app.
